samples/cellular/udp/src/main.c

```diff
@@ -105,6 +136,23 @@ static int modem_init(void)
 		return err;
 	}
 
+#if defined(CONFIG_UDP_RAI_ENABLE)
+	/* Enable Access Stratum RAI support.
+	 * Note: The 1.3.x modem firmware release is certified to be compliant with 3GPP Release 13.
+	 * %REL14FEAT enables selected optional features from 3GPP Release 14. The 3GPP Release 14
+	 * features are not GCF or PTCRB conformance certified by Nordic and must be certified
+	 * by MNO before being used in commercial products.
+	 * Available on nRF9160, unavailable on nRF9161 (features supported by default).
```

Comment:
I would rephrase this a bit. With nRF9161 the AS-RAI feature support is always enabled and cannot be disabled. Because of this I think saying "by default" is not correct, because there's no way to change the setting like with nRF9160. For example:
```suggestion
	/* Enable Access Stratum RAI support for nRF9160.
	 * Note: The 1.3.x modem firmware release is certified to be compliant with 3GPP Release 13.
	 * %REL14FEAT enables selected optional features from 3GPP Release 14. The 3GPP Release 14
	 * features are not GCF or PTCRB conformance certified by Nordic and must be certified
	 * by MNO before being used in commercial products.
	 * nRF9161 is certified to be compliant with 3GPP Release 14.
```